diff --git a/node_modules/react-native-thermal-receipt-printer/android/build.gradle b/node_modules/react-native-thermal-receipt-printer/android/build.gradle
index 900912a..e18ea56 100644
--- a/node_modules/react-native-thermal-receipt-printer/android/build.gradle
+++ b/node_modules/react-native-thermal-receipt-printer/android/build.gradle
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion = 29
-    buildToolsVersion = "29.0.2"
+    compileSdkVersion = 35
+    buildToolsVersion = "35.0.0"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 29
+        minSdkVersion 24
+        targetSdkVersion 35
         versionCode 1
         versionName "1.0"
 
@@ -26,7 +26,7 @@ dependencies {
     androidTestImplementation('androidx.test.espresso:espresso-core:3.1.0', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    api 'androidx.appcompat:appcompat:1.0.0'
+    api 'androidx.appcompat:appcompat:1.6.1'
     api "com.facebook.react:react-native:+"
     testImplementation 'junit:junit:4.12'
 }
